<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Dlg</class>
 <widget class="QDialog" name="Main_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>食品安全快速检测仪</string>
  </property>
  <widget class="QPushButton" name="Btn_FoodTest">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>食品安全检测</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/new/prefix1/Ico/food.ico</normaloff>:/new/prefix1/Ico/food.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL ShanHeiSun Uni [Misc]</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>食品安全快速检测仪</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_SpectralScan">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>光谱扫描</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/new/prefix3/Ico/bo.ico</normaloff>:/new/prefix3/Ico/bo.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_QuanMeasure">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>定量测量</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/new/prefix2/Ico/ren.ico</normaloff>:/new/prefix2/Ico/ren.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_TimeSweep">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时间扫描</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/new/prefix4/Ico/time.ico</normaloff>:/new/prefix4/Ico/time.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_SystemSet">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>360</y>
     <width>441</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/new/prefix6/Ico/systemset.ico</normaloff>:/new/prefix6/Ico/systemset.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QProgressBar" name="BatteryPower">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>119</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
